src/workflows/tests/vervet-resolver.test.ts

```diff
@@ -0,0 +1,67 @@
+import * as fs from "fs";
+import * as path from "path";
+import * as os from "os";
+import { loadVervetResoucePaths } from "../vervet-resolver";
+
+describe("vervet init", async () => {
+  const originalwd = process.cwd();
+  var tmpDir: string = "";
+
+  beforeEach(() => {
+    tmpDir = fs.mkdtempSync(path.join(os.tmpdir(), "foo-"));
+    fs.writeFileSync(
+      path.join(tmpDir, ".vervet.yaml"),
+      `
+apis:
+  testapi:
+    resources:
+      - path: 'src/testapi/something/something/resources'
+      - path: 'src/testapi/something-else/something-else/resources'
+  testapi2:
+    resources:
+      - path: 'src/testapi2/resources'
+`,
+    );
+  });
+
+  afterEach(() => {
+    process.chdir(originalwd);
+    if (tmpDir !== "") {
+      fs.rmSync(tmpDir, { force: true, recursive: true });
+    }
+  });
+
+  it("chooses the first resource path in the first API", async () => {
+    // arrange
+    process.chdir(tmpDir);
+    // act
+    const vervetPaths = await loadVervetResoucePaths();
+    // assert
+    expect(vervetPaths?.defaultResourcesPath).toMatch(
+      new RegExp(".*src/testapi/something/something/resources$"),
+    );
+  });
+
+  it("works from a subdirectory in the same project", async () => {
+    // arrange
+    const otherDir = path.join(tmpDir, "other-place");
+    fs.mkdirSync(otherDir);
+    process.chdir(otherDir);
+    // act
+    const vervetPaths = await loadVervetResoucePaths();
+    // assert
+    expect(vervetPaths?.defaultResourcesPath).toMatch(
+      new RegExp(".*src/testapi/something/something/resources$"),
+    );
+  });
+
+  it("requires a vervet config", async () => {
+    // arrange
+    process.chdir(tmpDir);
+    fs.unlinkSync(".vervet.yaml");
+    // act
+    const vervetPaths = await loadVervetResoucePaths();
+    // assert
+    expect(vervetPaths).toBeFalsy();
+  });
```

Comment:
Is it possible to add tests for the cases in vervet-resolver when the vervet config is invalid? 
- when the yaml doesn't contain 'apis'
- when the yaml doesn't contain 'resources'